cmd/jaeger/ingester-with-remote.yaml

```diff
@@ -0,0 +1,31 @@
+service:
+  extensions: [jaeger_storage]
+  pipelines:
+    traces:
+      receivers: [kafka]
+      processors: [batch]
+      exporters: [jaeger_storage_exporter]  # same as in the default cmd/jaeger-v2/config.yaml
+  telemetry:
+    metrics:
+      address: 0.0.0.0:8889 # to avoid port conflict with collector-with-kafka.yaml
+
+extensions:
+  jaeger_storage:
+    grpc:
+      memstore:
+        server: localhost:17271
+        connection-timeout: 5s
+
+receivers:
+  kafka:
+    brokers:
+      - localhost:9092
+    encoding: otlp_proto # available encodings are otlp_proto, jaeger_proto, jaeger_json, zipkin_proto, zipkin_json, zipkin_thrift
+    initial_offset: earliest # consume messages from the beginning
```

Comment:
perhaps we should parameterize this and default to latest checkpoint rather than earliest, since it would be very bad to run with earliest in production. The integration tests can override the value via env var.